.devcontainer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM mcr.microsoft.com/devcontainers/base:ubuntu-20.04
+
+RUN apt-get update && apt-get install -y sudo
+
+# Install Node LTS
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash -
+RUN sudo apt-get install -y nodejs
+# Install MongoDB
+# Source: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu-tarball/#install-mongodb-community-edition
+RUN mkdir -p /tmp/mongodb && \
+    cd /tmp/mongodb && \
+    wget -qOmongodb.tgz https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu2004-5.0.2.tgz && \
+    tar xf mongodb.tgz && \
+    cd mongodb-* && \
+    sudo cp bin/* /usr/local/bin/ && \
+    rm -rf /tmp/mongodb && \
+    sudo mkdir -p /data/db && \
+    sudo chown vscode:vscode -R /data/db
```

Comment:
You can/should combine these in a single statement and make the image slightly smaller. Also, I would use a couple of variables up top to set the versions for MongoDB and Nodejs. That should make the maintenance slightly better.